bootstrap/velocity/src/main/java/org/geysermc/geyser/platform/velocity/GeyserVelocityPingPassthrough.java

```diff
@@ -54,18 +54,30 @@ public GeyserPingInfo getPingInformation(InetSocketAddress inetSocketAddress) {
         } catch (ExecutionException | InterruptedException e) {
             throw new RuntimeException(e);
         }
+
+        // Apparently, plugins such as MiniMOTD can just yeet all players
+        int maxPlayers;
+        int onlinePlayers;
+        if (event.getPing().getPlayers().isPresent()) {
+            maxPlayers = event.getPing().getPlayers().get().getMax();
+            onlinePlayers = event.getPing().getPlayers().get().getOnline();
+        } else {
+            maxPlayers = 1; // Gotta fallback to something
+            onlinePlayers = 0;
+        }
+
         GeyserPingInfo geyserPingInfo = new GeyserPingInfo(
                 LegacyComponentSerializer.legacy('§').serialize(event.getPing().getDescriptionComponent()),
-                new GeyserPingInfo.Players(
-                        event.getPing().getPlayers().orElseThrow(IllegalStateException::new).getMax(),
-                        event.getPing().getPlayers().orElseThrow(IllegalStateException::new).getOnline()
-                ),
+                new GeyserPingInfo.Players(maxPlayers, onlinePlayers),
```

Comment:
instead of adding a whole new if-else block, use `Optional#map(...)` to convert it to an optional of the player counts, and then use `orElse(<0 or 1>)`

Reply:
like this https://github.com/GeyserMC/Geyser/pull/3930/files#diff-74e0ca9af638237203871557a54d9077a0d6b83877650880139161ff78c083c1R59

Reply:
Seeing how the other PR adds that fix anyways, i'll wait until it is merged and then rebase on that :p
Thanks for the link :)